twitter/__main__.py

```diff
@@ -0,0 +1,35 @@
+import click
+
+from . import fetch
+from .api import create_api
+from .mongo import create_client
+from .store import store
+
+
+@click.command()
+@click.argument("filename", type=click.Path(exists=True, dir_okay=False))
+@click.option("--limit", "-l", type=int)
+@click.option("--count", "-c", type=int, default=50)
+@click.option("--database", default="default")
+@click.option("--collection", default="test")
+def main(filename, limit: int, count: int, database: str, collection: str):
```

Comment:
I get a syntax error on `limit: int` when running `python -m twitter path/to/tweets.csv`